### DISCUSSION TITLE
Versioning the DCP

### DISCUSSION THREAD

**Discussion by arnoweiss** [eclipse-dataspace-dcp/decentralized-claims-protocol#128] [General]
The current release is tagged 0.8.1. The context URI is `https://w3id.org/dspace-dcp/v0.8` holding the namespace `https://w3id.org/dspace-dcp/v0.8/`. Let's think about what that should look like for an upcoming stable release. Introducing that asap would make sense so there's stability of URIs across editor's drafts.

Suggestion:

tag: `1.0.0-RC1`
context uri: `https://w3id.org/dspace-dcp/v1.0/dcp.jsonld`
namespace uri: `https://w3id.org/dspace-dcp/v1.0/`

cc @paullatzelsperger @wolf4ood

**Comment by wolf4ood** [comment]
> context uri: https://w3id.org/dspace-dcp/v1.0/dcp.jsonld
namespace uri: https://w3id.org/dspace-dcp/v1.0/

Sounds good to me
